src/main/webapp/app/entities/submission/submission.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { SERVER_API_URL } from 'app/app.constants';
+
+import * as moment from 'moment';
+
+import { createRequestOption } from 'app/shared';
+import { Result } from 'app/entities/result';
+import { Submission } from 'app/entities/submission';
+import { Participation } from 'app/entities/participation';
+
+export type EntityResponseType = HttpResponse<Submission>;
+export type EntityArrayResponseType = HttpResponse<Submission[]>;
+
+@Injectable({ providedIn: 'root' })
+export class SubmissionService {
```

Comment:
We need some simple jest tests for this service that verify that the correct endpoint is called.
We have some examples like this as e.g. src/test/javascript/spec/service/programming-exercise-test-case-service.spec.ts.

Reply:
done :)